src/QMCTools/convert4qmc.cpp

```diff
@@ -55,10 +55,23 @@ int main(int argc, char **argv)
   QMCGaussianParserBase *parser=0;
   int iargc=0;
   std::string in_file(argv[1]);
+
+  
   std::string punch_file;
   std::string psi_tag("psi0");
   std::string ion_tag("ion0");
-  std::string prefix("sample");
+  std::string prefix(argv[2]);
+
+
+  std::string delimiter =".out";
+  int pos = 0;
+  std::string token;
+  pos = prefix.find(delimiter);
+  token = prefix.substr(0, pos);
+  prefix.erase(0, pos + delimiter.length());
+  prefix=token;
+  std::cout << "PREFIX="<<prefix << std::endl;
```

Comment:
Friendlier formatting "Using " << prefix << " to name output files"

Reply:
Done